content/github/collaborating-with-issues-and-pull-requests/about-pull-requests.md

```diff
@@ -53,6 +53,10 @@ When you create a pull request, you can choose to create a pull request that is
 
 {% data reusables.pull_requests.mark-ready-review %} You can convert a pull request to a draft at any time. For more information, see "[Changing the stage of a pull request](/articles/changing-the-stage-of-a-pull-request)."
 
+### Difference between commits on Compare page and Pull Request page
+
+Compare pages show a compare between the tip of the head ref and the current common ancestor (i.e. merge base) of the head and base ref. On the other hand, pull request pages show a compare between the tip of the head ref and the common ancestor of the head and base ref at the time when the pull request was created. In other words, the merge base used for the comparison might be different.
```

Comment:
Changes suggested to make this easier for people to read if English isn't their first language.

```suggestion
The compare and pull request pages use different methods to calculate the diff for changed files:

- Compare pages show the diff between the tip of the head ref and the current common ancestor (that is, the merge base) of the head and base ref. 
- Pull request pages show the diff between the tip of the head ref and the common ancestor of the head and base ref at the time when the pull request was created. Consequently, the merge base used for the comparison might be different.
```